lib/BackgroundJob/Tasks/CreateClustersTask.php

```diff
@@ -200,6 +200,10 @@ private function createClusterIfNeeded(string $userId) {
 		$mergedClusters = $this->mergeClusters($currentClusters, $newClusters);
 		$this->personMapper->mergeClusterToDatabase($userId, $currentClusters, $mergedClusters);
 
+		// Remove all orphaned persons (those without any faces)
+		// NOTE: we will do this for all models, not just for current one, but this is not problem.
+		$this->personMapper->deleteOrphaned($userId);
```

Comment:
Why not print a warning when delete orphaned persons?
Hopefully it will never be printed. but if it happens, we can analyze it ..
Just:
`if ($this->personMapper->deleteOrphaned($userId))`
or 
`if (count($this->personMapper->deleteOrphaned($userId))`